coderd/database/migrations/000219_notifications.down.sql

```diff
@@ -0,0 +1,5 @@
+DROP TABLE IF EXISTS notification_messages;
+DROP TABLE IF EXISTS notification_templates;
+DROP TYPE IF EXISTS notification_receiver;
+DROP TYPE IF EXISTS notification_method;
+DROP TYPE IF EXISTS notification_message_status;
```

Comment:
Can you drop enums safely? I thought that there are some risks around it.

Reply:
Yup, I tested this again now and it works.